File: VB/DevExpress.Example03/Employee.vb
Imports DevExpress.Mvvm

Namespace DevExpress.Example03

    Public Enum Gender
        Male
        Female
    End Enum

    Public Class Employee
        Inherits BindableBase

        Protected _Name As String

        Public Property Name As String
            Get
                Return _Name
            End Get

            Set(ByVal value As String)
                SetProperty(_Name, value, "Name")
            End Set
        End Property

        Protected _Age As Integer

        Public Property Age As Integer
            Get
                Return _Age
            End Get

            Set(ByVal value As Integer)
                SetProperty(_Age, value, "Age")
            End Set
        End Property

        Protected _Gender As Gender

        Public Property Gender As Gender
            Get
                Return _Gender
            End Get

            Set(ByVal value As Gender)
                SetProperty(_Gender, value, "Gender")
            End Set
        End Property

        Protected _Department As String

        Public Property Department As String
            Get
                Return _Department
            End Get

            Set(ByVal value As String)
                SetProperty(_Department, value, "Department")
            End Set
        End Property

        Protected _IsInvited As Boolean

        Public Property IsInvited As Boolean
            Get
                Return _IsInvited
            End Get

            Set(ByVal value As Boolean)
                SetProperty(_IsInvited, value, "IsInvited")
            End Set
        End Property
    End Class
End Namespace

File: CS/DevExpress.Example03/Employee.cs


using DevExpress.Mvvm;

namespace DevExpress.Example03 {

    public enum Gender { Male, Female }
    
    public class Employee : BindableBase {

        protected string _Name;

        public string Name {
            get {
                return this._Name;
            }

            set {
                this.SetProperty(ref this._Name, value, "Name");
            }
        }
        
        protected int _Age;

        public int Age {
            get {
                return this._Age;
            }

            set {
                this.SetProperty(ref this._Age, value, "Age");
            }
        }

        protected Gender _Gender;

        public Gender Gender {
            get {
                return this._Gender;
            }

            set {
                this.SetProperty(ref this._Gender, value, "Gender");
            }
        }

        protected string _Department;

        public string Department {
            get {
                return this._Department;
            }

            set {
                this.SetProperty(ref this._Department, value, "Department");
            }
        }

        protected bool _IsInvited;

        public bool IsInvited {
            get {
                return this._IsInvited;
            }

            set {
                this.SetProperty(ref this._IsInvited, value, "IsInvited");
            }
        }
    }
}
